processus-traitement-donnees.md

```diff
@@ -192,29 +191,44 @@ Par exemple, si certaines données n'ont pas changé d'une période sur l'autre,
 
 Le compactage se lance avec la commande suivante:
 
-```
-http :3000/api/data/compact [options]
+```sh
+cd opensignauxfaibles
+./sfdata compact [options]
 # Par exemple
-http :3000/api/data/compact fromBatchKey="1804"
+./sfdata compact --fromBatchKey="1804"
 ```
 
 L'option `fromBatchKey` indique le premier batch dans l'ordre alphabétique qui nécessite d'être compacté (c'est-à-dire qui a subi des changements). Tous les suivants le seront automatiquement.
 
-> Important: Le compactage est une opération difficilement réversible. Pour prévenir toute corruption de données et/ou interruption prématurée du compactage, nous recommandons de valider les données importées avant leur compactage, à l'aide de `http :3000/api/data/validate collection="ImportedData"`. (cf [Procédure d'importation de données](procedure-import-donnees.md))
+> Important: Le compactage est une opération difficilement réversible. Pour prévenir toute corruption de données et/ou interruption prématurée du compactage, nous recommandons de valider les données importées avant leur compactage, à l'aide de `./sfdata validate --collection="ImportedData"`. (cf [Procédure d'importation de données](procedure-import-donnees.md))
 
 ## Spécificités des calculs de variables
 
 TODO
 `param` dans la collection Admin
 
-Le calcul des variables est lancé via la requête:
+Le calcul des variables est lancé de la manière suivante:
 
+```sh
+cd opensignauxfaibles
+./sfdata reduce [options]
+# Par exemple
+./sfdata reduce --up-to-batch="1904" --key="01234567891011"
```

Comment:
Si `parsers="all"` t'a plu, peut être que `--key="all"` serait pas mal aussi ?

Reply:
Même réponse: `key` est optionnel, et sert surtout pour le déboggage => je vais rendre ça plus clair.

Ça me fait penser que, pour coller à `--since-batch`, `--up-to-batch` gagnerait à être renommé en `--until-batch`.

Reply:
=> cf https://github.com/signaux-faibles/documentation/pull/31/commits/cab646fcc9439680cce363995cb98fd0ccbe8657 et https://github.com/signaux-faibles/documentation/pull/31/commits/a29afd0b124a88b25a6601c1cd62d376062df551